src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -235,8 +226,10 @@ public JSONObject getJenkinsBuildStatusData(String jobName, Integer buildNumber)
         dataRecieved = response.getStatusCode() == HttpStatus.SC_OK;
 
         if (!dataRecieved) {
-            String message = "Failed to job data from job " + jobName + " and build " + buildNumberString
-                    + ". Status code: " + response.getStatusCodeValue() + ". Possible not built yet.";
+            String message = "Failed to job data from job " + jobName + " and build "
```

Comment:
Failed to ... what? I'm missing a verb here 😄

Reply:
"Failed to get status data" I suppose.